src/common/GitHubVS.ruleset

```diff
@@ -37,6 +37,7 @@
         <Rule Id="CA1501" Action="None" />
         <Rule Id="CA1502" Action="None" />
         <Rule Id="CA1506" Action="None" />
+        <Rule Id="CA1701" Action="None" />
```

Comment:
Yeah, this is a good idea, those conditions won't trigger for our other usages of string literals (logging and such), so should be helpful!